datadotworld/datadotworld.py

```diff
@@ -182,6 +182,7 @@ def load_dataset(self, dataset_key, force_update=False):
                                                   '%Y-%m-%dT%H:%M:%S.%fZ')
                 if (last_modified > datetime.utcfromtimestamp(
                         path.getmtime(str(descriptor_file)))):
+                    simplefilter('always', UserWarning)
```

Comment:
The match here is too broad. Our package should not change the warning filter for `UserWarning` globally. I'm ok with always showing this warning, but we should match on message or line number, to make it more specific.

Reply:
Addressed. I matched the warning to always show by matching the message.